src/Compute/Compute/Generated/Models/PSAccessUri.cs

```diff
@@ -29,5 +29,7 @@ public partial class PSAccessUri
     {
         public string AccessSAS { get; set; }
 
+        public string SecurityDataAccessSAS { get; private set; }
```

Comment:
AccessSAS also does not have private despite having it in SDK. Keep it consistent for now, let users modify PS objects, SDK won't let them modify the others.